data/districts.json

```diff
@@ -56,7 +56,7 @@
   },
   {
-    "name": "Tehrathum",
```

Comment:
The word actually matches the Wiki information.

https://en.wikipedia.org/wiki/Tehrathum_District

Fixing typo can lead it to make the project maintain harder because Roman Nepali doesn't have constant names.

So, I think we should use Wiki as reference to keep things constant at least for a while unless we find proper solution.